episodes/filetypes-and-metadata.md

```diff
@@ -0,0 +1,498 @@
+---
+title: 'Filetypes and metadata'
+teaching: 30
+exercises: 10
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- Which file formats should be used for microscopy images?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Explain the pros and cons of some popular image file formats
+- Explain the difference between lossless and lossy compression
+- Inspect image metadata with the napari-aicsimageio plugin
+- Inspect and set an image's scale in Napari
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## Image file formats
+
+Images can be saved in a wide variety of file formats - e.g. you may be familiar 
+with some like .jpg, .png or .tiff. Microscopy images have an especially wide 
+range of options, with hundreds of different formats in use, often specific to 
+particular microscope manufacturers. With this being said, how do we choose 
+which format to use?
+
+## Metadata
+
+First, let's look closer at what gets stored inside an image file.
+
+There are two main things that get stored inside image files: _pixel values_ and 
+_metadata_. We've looked at pixel values in previous episodes -  this is the raw 
+image data as an array of numbers with specific dimensions and data type. The 
+metadata, on the other hand, is a wide variety of additional information about 
+the image and how it was acquired.
+
+For example, let's take a look at the metadata in the 
+'Plate1-Blue-A-12-Scene-3-P3-F2-03.czi' file we downloaded as part of the 
+[setup instructions](../learners/setup.md). To browse the metadata we will use 
+a Napari plugin called [napari-aicsimageio
+](https://www.napari-hub.org/plugins/napari-aicsimageio). This plugin allows a 
+wide variety of file formats to be opened in Napari that aren't supported by 
+default. This plugin was already installed in the [setup instructions
+](../learners/setup.md), so you should be able to start using it straight away.
+
+:::::::::::::::::::::::::::::::::::::: callout
+
+### Napari-aicsimagio installation
```

Comment:
I'm not sure that this is the right place for this callout. Would it be better to have it in the setup instructions? Here it I think it distracts from the episodes learning objectives.

Reply:
Sounds good - let's move this to the setup instructions then. @thompson318 - could you add this to your setup instructions PR?

Reply:
Just checked, this is done in setup.